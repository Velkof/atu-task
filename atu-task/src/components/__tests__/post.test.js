import React from 'react';
import renderer from 'react-test-renderer';
import Post from '../post';

const post = {
    id: 1,
    title: 'Post title',
    userId: 2,
    body: 'Post body'
}

test('<Post /> renders successfully', () => {
  const tree = renderer.create(
    <Post
        key={post.id}
        {...post}
        onClick={() => {}}
    />
  ).toJSON();
  
  expect(tree).toMatchSnapshot();
});
