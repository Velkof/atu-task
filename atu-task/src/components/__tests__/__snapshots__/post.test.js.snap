// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Post /> renders successfully 1`] = `
<div
  className="post"
>
  <div
    className="post-id"
  >
    <p>
      1
    </p>
  </div>
  <div
    className="info-container"
  >
    <p
      className="title"
    >
      Post title
    </p>
  </div>
  <div
    className="button-container"
  >
    <button
      onClick={[Function]}
    >
      Read post
    </button>
  </div>
</div>
`;
